src/test/resources/Contacts5kSql_avro_schema.json

```diff
@@ -1,50 +1 @@
-{
-  "type": "record",
-  "name": "topLevelRecord",
-  "fields": [
-    {
-      "name": "row_id",
-      "type": [
-        "null",
-        "int"
-      ],
-      "doc": ""
-    },
-    {
-      "name": "person_name",
-      "type": [
-        "null",
-        {
-          "type": "string",
-          "logicalType": "varchar",
-          "maxLength": 65535
-        }
-      ],
-      "doc": ""
-    },
-    {
-      "name": "contact_type",
-      "type": [
-        "null",
-        {
-          "type": "string",
-          "logicalType": "varchar",
-          "maxLength": 65535
-        }
-      ],
-      "doc": ""
-    },
-    {
-      "name": "contact_number",
-      "type": [
-        "null",
-        {
-          "type": "string",
-          "logicalType": "varchar",
-          "maxLength": 65535
-        }
-      ],
-      "doc": ""
-    }
-  ]
-}
+{"type":"record","name":"topLevelRecord","fields":[{"name":"row_id","type":["null","int"],"doc":""},{"name":"person_name","type":["null",{"type":"string","logicalType":"varchar","maxLength":16383}],"doc":""},{"name":"contact_type","type":["null",{"type":"string","logicalType":"varchar","maxLength":16383}],"doc":""},{"name":"contact_number","type":["null",{"type":"string","logicalType":"varchar","maxLength":16383}],"doc":""}]}
```

Comment:
can you pretty-format the JSON for better readability.
Extra size is ok as its a test resource.

Reply:
@posabogdanpetre can you make it pretty JSON

Reply:
Fixed it @anantdamle